metaflow/tutorials/02-statistics/stats.py

```diff
@@ -57,10 +58,10 @@ def start(self):
         # We want to compute some statistics for each genre. The 'foreach'
         # keyword argument allows us to compute the statistics for each genre in
         # parallel (i.e. a fan-out).
-        self.next(self.compute_statistics, foreach="genres")
+        self.next(self.computestatistics, foreach="genres")
```

Comment:
this shouldn't be needed

Reply:
Also, can we not use pandas in this file? pandas version mismatch between this file and the subsequent ones has bitten us plenty of times.

Reply:
I will get rid of pandas. The reason for the name change was for argo workflows, which caused issues with underscores. Do you know another way around that failure (seems like an Argo issue)?

Reply:
All external packages are removed from lesson 02 code now